src/partials/page-analyse.html

```diff
@@ -193,13 +193,13 @@ <h3 class="analyseBoard-title">{{title}}</h3>
 							{{/if}}
 
 							<button class="btn analyseBoard-iconBtn analyseBoard-expand" data-id="chart{{@index}}">
-								<img src="{{../root}}assets/img/icons/expand.svg" aria-label="{{expand}}">
-								<img src="{{../root}}assets/img/icons/collapse.svg" aria-label="{{collapse}}">
+								<img src="{{../root}}assets/img/icons/expand.svg" alt="" aria-hidden="true">
```

Comment:
Buttons are now completely unusable by blind people. Either add an aria-label to the <button> or add a .sr-only <span> with a description inside